File: stepik-union/src/main/java/org/stepik/plugin/refactoring/impl/pycharm/AcceptableClasses.java
package org.stepik.plugin.refactoring.impl.pycharm;

import com.intellij.psi.PsiElement;
import com.intellij.psi.PsiFileSystemItem;
import com.jetbrains.python.psi.PyClass;
import org.jetbrains.annotations.NotNull;

import java.util.HashSet;
import java.util.Set;

/**
 * @author meanmail
 */
public class AcceptableClasses {
    @NotNull
    public static Set<Class<? extends PsiElement>> get() {
        Set<Class<? extends PsiElement>> acceptableClasses = new HashSet<>();

        acceptableClasses.add(PsiFileSystemItem.class);
        acceptableClasses.add(PyClass.class);

        return acceptableClasses;
    }
}

File: stepik-union/src/main/kotlin/org/stepik/plugin/refactoring/impl/pycharm/AcceptableClasses.kt
package org.stepik.plugin.refactoring.impl.pycharm

import com.intellij.psi.PsiElement
import com.intellij.psi.PsiFileSystemItem
import java.util.*


object AcceptableClasses {
    fun get(): Set<Class<out PsiElement>> {
        val acceptableClasses = HashSet<Class<out PsiElement>>()

        acceptableClasses.add(PsiFileSystemItem::class.java)

        return acceptableClasses
    }
}
